packages/desktop-client/README.md

```diff
@@ -32,11 +32,12 @@ Prerequisites:
 
 #### Running against the local server
 
-First start the dev server:
+First start a dev instance:
```

Comment:
I changed this to not use the word "server" since we are not using acutal-server here, just the app repo.